invenio_users_resources/records/mappings/os-v2/users/user-v3.0.0.json

```diff
@@ -0,0 +1,206 @@
+{
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "default": {
+          "type": "custom",
+          "tokenizer": "uax_url_email",
+          "filter": [
+            "lowercase"
+          ]
+        },
+        "default_search": {
+          "type": "custom",
+          "tokenizer": "uax_url_email",
+          "filter": [
+            "lowercase"
+          ]
+        },
+        "edge_analyzer": {
+          "tokenizer": "uax_url_email",
```

Comment:
is the idea here to keep the email as an unique token, and then use `edgegrams` to match the beginning of the email?
Example: mario.rossi@gmail.com
If I start typing `rossi`, do I find him?

Reply:
No, you would not find him. However I think this is probably the behaviour we want, in terms of trade off between size of index and allowing users to search effectively